src/WorkItems/AzureDevOpsFilingClient.cs

```diff
@@ -20,16 +20,20 @@ namespace Microsoft.WorkItems
     /// </summary>
     public class AzureDevOpsFilingClient: FilingClient
     {
-        private WorkItemTrackingHttpClient _witClient;
+        internal IWorkItemTrackingHttpClient _witClient;
```

Comment:
>_witClient [](start = 45, length = 10)

If you're going to expose it, let's lose the underscore. FWIW I'd even capitalize it.

Reply:
Disagree. I actually prefer in unit-tests to see the underscore so that I know there that we're digging into what should be a private field (as opposed to something actually exposed to other consumers).

---
In reply to: [389389947](https://github.com/microsoft/sarif-sdk/pull/1800#discussion_r389389947) [](ancestors = 389389947)